apps/L1G/tinder/tinder-frontend/specs/MultiSelect.spec.tsx

```
@@ -0,0 +1,148 @@
+import { MultiSelect } from '../src/components/MultiSelect';
+import '@testing-library/jest-dom';
+
+const mockOptions = [
+  { value: 'music', label: 'Music' },
+  { value: 'sports', label: 'Sports' },
+  { value: 'movies', label: 'Movies' },
+  { value: 'books', label: 'Books' },
+];
+
+class ResizeObserver {
+  observe() {}
```

ESLint

[error] Unexpected empty method 'observe'.

(@typescript-eslint/no-empty-function)